contracts/Vault.vy

```diff
@@ -102,7 +102,7 @@ struct StrategyParams:
 event StrategyAdded:
     strategy: indexed(address)
     debtLimit: uint256  # Maximum borrow amount
-    rateLimit: uint256  # Increase/decrease per block
+    rateLimit: uint256  # Max increase in debt per second since last harvest
```

Comment:
"Limit on the increase of debt per unit time since last harvest"

Reply:
fixed in latest